recipes/dcw-gmt/bld.bat

```diff
@@ -0,0 +1,8 @@
+set DATADIR="%LIBRARY_PREFIX%\share\dcw-gmt"
+
+if not exist %DATADIR% mkdir %DATADIR%
+
+xcopy %SRC_DIR%\*.txt %DATADIR%
+xcopy %SRC_DIR%\*.TXT %DATADIR%
```

Comment:
Windows is not case sensitive, so you can remove one of those `*.txt` or `*.TXT`.